src/main/java/com/volunteer/enterprise/dao/VolunteerDAOStub.java

```diff
@@ -23,40 +21,17 @@ public Volunteer save(Volunteer volunteer) throws Exception {
     }
     @Override
     public List<Volunteer> fetchAll() {
-        HashMap<Object, Object> allVolunteer;
         List<Volunteer> returnVolunteers = new ArrayList(allVolunteers.values());
         return returnVolunteers;
     }
 
     @Override
     public Volunteer fetch(int id) {
-        return null;
+        return allVolunteers.get(id);
     }
 
     @Override
     public void delete(int id) {
         allVolunteers.remove(id);
     }
-    
-        @Override
-    public Company save(Volunteer company) throws Exception {
-        allCompanies.put(company.getCompanyID(),company);
-        return company;
-    }
-    @Override
-    public List<Company> fetchAll() {
-        HashMap<Object, Object> allCompany;
-        List<Company> returnCompanies = new ArrayList(allCompanies.values());
-        return returnCompanies;
-    }
-
-    @Override
-    public Company fetch(int id) {
-        return null;
-    }
-
-    @Override
-    public void delete(int id) {
-        allCompanies.remove(id);
-    }
```

Comment:
Removed code meant for `CompanyDAO` class from `VolunteerDAO` class/file to separate `CompanyDAO` file.